code-styling/mysql/README.md

```diff
@@ -347,11 +446,12 @@ LEFT JOIN table2 AS t2
 
 ### Delete queries
 
-Delete queries SHOULD not be used.
+Delete queries SHOULD NOT be used. Instead a column to indicate that an item is
```

Comment:
The whole previous discussion here could be summarized as 'this has nothing to do with sql styling, table design or even sql in general, but everything with the data model/storage' and thus it should not be in this document in my opinion.
The whole 'throw nothing away' gave us a big mess, lets learn from that and not document 'deleting data is bad'